Add Media preload defaults to metadata Sample

beaufortfrancois opened the pull request:
@japacible @mounirlamouri,

Here's the sample for [Media preload defaults to metadata](https://www.chromestatus.com/feature/5682169347309568).

If that looks good to you, I'll merge it once https://chromium-review.googlesource.com/c/chromium/src/+/596623 is pushed.

![screenshot 2017-08-23 at 3 30 08 pm](https://user-images.githubusercontent.com/634478/29618284-f5f467a2-8817-11e7-9b32-dcae79a13211.png)

beaufortfrancois committed:
Add Media preload defaults to metadata Sample

wolenetz commented:
Drive-by comments:

The preload attribute's "hint" is ignored per spec when a MediaSource is attached to the media element (https://www.w3.org/TR/media-source/#mediasource-attach). This might deserve mention in this sample. WDYT?

@japacible @mounirlamouri Similar note might be a good thing to include in https://www.chromestatus.com/feature/5682169347309568

beaufortfrancois committed:
Add MSE note

beaufortfrancois commented:
Thanks @wolenetz!

Fixed in https://github.com/GoogleChrome/samples/pull/539/commits/d5a3f5308cc8ddc6de4b40b21427324aa01b5f1f

I've also updated https://www.chromestatus.com/feature/5682169347309568 and filed https://github.com/GoogleChrome/chromium-dashboard/issues/496

mounirlamouri commented:
lgtm :+1:

beaufortfrancois commented:
I'll merge for now and will update sample availability if it does not land in M62. Thank you folks!

beaufortfrancois commented:
Try it live at https://googlechrome.github.io/samples/media/preload-metadata in Chrome Canary with `--enable-features=PreloadDefaultIsMetadata`.

Note that https://www.chromestatus.com/feature/5682169347309568 has been updated as well.
